src/server/api/routers/webhook.ts

```diff
@@ -3,54 +3,97 @@ import { z } from "zod";
 import { env } from "~/env.mjs";
 import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
 
+interface GitHubAPIError {
+  status: number;
+}
+
 export const webhookRouter = createTRPCRouter({
-  createWebhook: protectedProcedure.input(z.object({ accessToken: z.string()})).mutation(async ({ input }) => {
+  // accessToken is optional because there can be instances where the user's session is remembered (automatically is redirected to hompage) 
+  // which makes the accessToken undefined since accessToken is grabbed when user is loggin in (authenticating).
+  // if this occurs, we use the accessToken in the DB and if that is stale, we require the user to log out and log back in.
+  createWebhook: protectedProcedure.input(z.object({ accessToken: z.string().optional() })).mutation(async ({ ctx, input }) => {
     const owner = "GabrielPedroza";
     const repo = "exotica";
 
     const WEBHOOK_URL = "https://hkdk.events/G3KE7hkFpr5I";
     // const WEBHOOK_URL = "https://bread-meta.vercel.app/api/webhooks/webhook";
-    
+
+    // grabbing DB access token
+    const accessToken = await ctx.prisma.account.findFirst({
+    where: {
```

Comment:
this indentation looks funny - did you run the formatter?

Reply:
yeah some are weird but after these 2 pr's are merged, i'll have another pr just for formatting with prettier so everything is more consistent.